lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -1542,6 +1543,38 @@ LinearLayout chooseScaledMfmaScaleLayout(
   return newLL;
 }
 
+// Create a LinearLayout similar to mfmaLayout where each thread holds 8
+// consecutive elements.
+LinearLayout chooseMfma8Layout(AMDMfmaEncodingAttr mfmaLayout,
+                               ArrayRef<int64_t> shape) {
+  assert(shape.size() == 2 && mfmaLayout.getMDim() == 32 &&
+         mfmaLayout.getNDim() == 32 && mfmaLayout.getIsTransposed());
+
+  MLIRContext *ctx = mfmaLayout.getContext();
+  StringAttr kRegister = S("register");
+  StringAttr kLane = S("lane");
+  StringAttr kWarp = S("warp");
+  StringAttr kBlock = S("block");
+
+  SmallVector<unsigned> order = {1, 0};
+  auto standardOutDims = standardOutDimNames(ctx, 2);
+  LinearLayout mfma8Layout = LinearLayout::empty();
+  mfma8Layout =
```

Comment:
Nit: put some comments here to explicitly mention that we delibrately make each thread handle 8 consecutive elements in order to get 128-bit global store for (b)f16 types, and keep threads in the lane follow the same pattern like canonical mfma layout.